.pipelines/v2/templates/job-test-project.yml

```diff
@@ -34,6 +37,30 @@ jobs:
     fetchDepth: 1
     fetchTags: false
 
+  - powershell: |
```

Comment:
In cases like this, when you are turning entire steps on and off, I would recommend using *compile time expressions* rather than `condition`s.

When you use a condition, it will show up in every iteration of the pipeline _including ones that do not use `usingLatestWebView2`._ This clutters the build logs.

If you use a compile-time expression, the steps will be deleted if they are not applicable:

```yaml
- ${{ if eq(parameters.usingLatestWebView2, true) }}:
  - powershell: |-
    your script here
  - script: |-
    your other script here
```

Reply:
Done. Thanks for your suggestion.